zbsprimarystorage/ansible/zbsp.py

```
@@ -115,7 +115,8 @@
 
 
 # name: make sure MDS has access to storage resources
-command = "[ -f /usr/local/bin/zbsadm ] && /usr/local/bin/zbsadm client deploy --host %s -p %s" % (host_post_info.host, host_post_info.remote_pass)
+command = ("[ -f /usr/local/bin/zbsadm ] && /usr/local/bin/zbsadm client deploy --host %s --port %s -u %s -p %s"
+           % (host_post_info.host, host_post_info.remote_port, host_post_info.remote_user, host_post_info.remote_pass))
 run_remote_command(command, host_post_info)
```

Ruff (0.12.2)

`run_remote_command` may be undefined, or defined from star imports

(F405)